packages/app-project/src/screens/ClassifyPage/ClassifyPage.js

```diff
@@ -71,55 +71,64 @@ function ClassifyPage({
     setShowTutorial(true)
   }
 
-  return (
-    <StandardLayout>
-      <Box
-        align='center'
-        gap='medium'
-        pad={{ horizontal: 'small', vertical: 'medium' }}
-      >
+  const [collectionsModalActive, setCollectionsModalActive] = useState(false)
```

Comment:
Declare all your hooks at the top of a functional component, before any conditional blocks.